bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: resolve errors in the private api docs

## Description
Fix errors in the private api docs.

## Verification Steps
1. Run `make run/docs`
    - Ensure request body example for `PUT /api​/managed-services-api​/v1​/agent-clusters​/{id}​/status` is rendered correctly.

![image](https://user-images.githubusercontent.com/9078522/115266270-fb80f700-a12f-11eb-8650-383de1e709b4.png)


## Type of change

## Checklist (Definition of Done)
